holidayextras/brands
# Responsive sizing for navbar-brand

#### What does this PR do? (please provide any background)

Mapped navbar-brand dimensions and positioning to the following responsive variables:
`@logo-width`
`@logo-height`
`@logo-width-sm`
`@logo-height-sm`
`@logo-width-md`
`@logo-height-md`
`@logo-width-lg`
`@logo-height-lg`

These variables are calculated based on the source image dimensions with these two variables:
`@logo-source-width`
`@logo-source-height`

Whilst making these changes, I took the liberty of appropriately reducing the size of the HX and hunter logos for mobile and tablet, and also improved the positioning of Airparks navbar-brand and BCP navbar-brand (mobile-only).

#### How can this be tested?
`cd pattern-library && npm link ../brands`

Resize your browser window and check the navbar-brand sizing/positioning for Airpakrs, BCP, HX and Hunter brands.

#### Any tech debt?

Other brands need to be updated to this pattern too - for now, I've just done the brands in Tripapp (+ Hunter).

#### Screenshots / Screencast
![image](https://cloud.githubusercontent.com/assets/2961244/24698113/723f0488-19e7-11e7-9ed8-5e6dda67561b.png)

#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/MtGAE4ZabBs4g/giphy.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
